### ISSUE TITLE
Missing arrow key text next to gif on beta site

### ISSUE THREAD

**Issue by Watson-B** [ezl/dailyidea.com#895]
On the beta site, the cards are missing the "Use arrow keys to see next idea" text next to the gif in the bottom right.

Beta Site:
![image](https://user-images.githubusercontent.com/65913664/108744177-7e018780-74ee-11eb-8787-16a97a5f496e.png)
Design:
<img width="1480" alt="card-arrow-text" src="https://user-images.githubusercontent.com/65913664/108744303-a4272780-74ee-11eb-9fc7-6787c6d20e07.png">

**Comment by vedmant** [comment]
@Watson-B This text instruction was removed by @ezl https://prnt.sc/106fsr3 not sure if I should put them back?

**Comment by Watson-B** [comment]
@vedmant Okay, I didn't know that. Let me check with him then.

**Comment by ezl** [comment]
@Watson-B  I removed that when I was working on the landing page.

I just made a snap judgment while working on it and decided:
(A)  icon itself would be sufficient
(B) the words take up a lot of room

I'm not married to it, so if you disagree, we can add it back. If it's not clear enough, we could try highlighting it more or putting an animation on it (though there are already a lot of moving elements on that page).

Will let you make the call on whether to add back in the text. Just let us know here.
